src/Orleans.Persistence.CosmosDB/CosmosDBGrainStorage.cs

```diff
@@ -432,7 +428,7 @@ private async Task TryCreateCosmosDBResources()
             for (var retry = 0; retry <= maxRetries; ++retry)
             {
                 var collResponse = await db.CreateContainerIfNotExistsAsync(
-                    stateCollection, offerThroughput);
+                    stateCollection, this._options.GetThroughputProperties());

```

Comment:
This was previously using the database level throughout, not sure if that has been raised or was correct in the first place. My only concern is that could be considered a breaking change? Although it might have been a small bug.